scripts/core/editor3/components/tables/TableCell.jsx

```diff
@@ -30,13 +30,35 @@ export class TableCell extends Component {
      * @description DraftJS key binding function.
      */
     keyBindingFn(e) {
+        if (e.ctrlKey && e.key === 'z') {
+            return 'parent-undo';
+        }
+
         if (e.ctrlKey && e.key === 'l') {
             return 'toggle-link';
         }
 
         return getDefaultKeyBinding(e);
     }
 
+    componentWillReceiveProps(nextProps) {
```

Comment:
`componentWillReceiveProps` is deprecated, use `UNSAFE_componentWillReceiveProps` if you really need it.

Reply:
UNSAFE_componentWillReceiveProps is available in 16.3. We are still using 16.2
There are another 14 places where componentWillReceiveProps is used so I think we have to fix all of them in a single PR.